File: backend/components/metiers/administration_publique/utils/validators/helpers/validators_helper.js
// validators_helper.js
// Helper validators JS pour Threed – exemple clé en main

/**
 * Vérifie si une chaîne est un email valide
 * @param {string} email
 * @returns {boolean}
 */
function isValidEmail(email) {
  return /^[^@\s]+@[^@\s]+\.[^@\s]+$/.test(email);
}

module.exports = { isValidEmail };

File: backend/components/metiers/threed/utils/validators/helpers/validators_helper.py
# validators_helper.py
# Helper validators Python pour Threed – exemple clé en main
import re

def is_valid_email(email):
    """
    Vérifie si une chaîne est un email valide
    """
    return bool(re.match(r'^[^@\s]+@[^@\s]+\.[^@\s]+$', email))
